### DISCUSSION TITLE
Add multitax or more 1

### DISCUSSION THREAD

**Discussion by reproteq** [InvoicePlane/InvoicePlane#1058] [Ideas]
is posible add more 1 tax to product? now i only  can put 1 tax normally in spain 21% but autonomo professional worker need add other tax irpf -15% in all products or jobs . thanks

**Comment by nielsdrost7** [comment]
- Add migration for a field on `invoice_items` and maybe on `quote_items`
- add fields to both forms (I think it's in the partials)
- save the form
- while saving the form, calculate the newly requested extra taxes
- recalculate totals, invoice_amounts, the works
- same for quotes
It's even more fun if you want to do it with ajax
